tools-local/targetgroups.props

```diff
@@ -2,13 +2,15 @@
 <Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <ItemGroup>
     <TargetGroups Include="netcore50">
+      <TargetFrameworkIdentifier>.NETCore</TargetFrameworkIdentifier>
```

Comment:
What about the `TargetFrameworkVersion` property?  Would it be better if we just set `TargetFramework` here instead, and let the SDK split it into TargetFrameworkIdentifier and TargetFrameworkVersion?

[https://github.com/dotnet/sdk/blob/33d51eab4d1e8dccf90eec946cf311ef29e0857f/src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.TargetFrameworkInference.targets#L49-L72](https://github.com/dotnet/sdk/blob/33d51eab4d1e8dccf90eec946cf311ef29e0857f/src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.TargetFrameworkInference.targets#L49-L72)

Reading the comment above that code, we may need to do the splitting ourselves for anything that isn't `.NETFramework`, `.NETCoreApp`, and `.NETStandard`.  (like for UAP, WindowsPhone, etc)

Reply:
I agree with @eerhardt here we either need to set this fully or just set TargetFramework and let the SDK set it.

Reply:
Right, makes sense. This will work as the Directory.Build.props file, which triggers the setconfiguration script generation target, is evaluated first.